instance/tasks.py

```diff
@@ -339,3 +344,111 @@ def cleanup_old_betatest_users():
         users_to_delete = queryset.filter(is_active=False, profile__modified__lte=delete_cutoff)
         logger.info('Deleting %s users.', users_to_delete.count())
         users_to_delete.delete()
+
+
+class KillZombiesRunner:
+    """
+    Helper class to run `kill_zombies_periodically`
+    """
+
+    def __init__(self):
+        self.region: str = getattr(settings, "OPENSTACK_REGION", "gra")
+        self.recipient: list = [email for _, email in getattr(settings, "ADMINS", [])]
+        self.threshold: int = getattr(settings, "KILL_ZOMBIES_WARNING_THRESHOLD")
```

Comment:
@spokerman12, using `getattr()` doesn't make much sense here unless you wanted to specify a default value and missed adding it. `settings.KILL_ZOMBIES_WARNING_THRESHOLD` should be okay otherwise.

Reply:
Set `10` as default.

Reply:
@spokerman12, defining `10` as a default here duplicates the default set in `settings.KILL_ZOMBIES_WARNING_THRESHOLD` and hence is unnecessary. So you should just use `settings.KILL_ZOMBIES_WARNING_THRESHOLD` here.

Reply:
Deleted

Reply:
My bad for overlooking this; it's too ingrained in my head that when fetching settings one should always set a default.

Reply:
Oh, and I also picked it up from here https://github.com/open-craft/opencraft/blob/f043e87a2fd3ac26cf3a7528d411d0ebda3b693b/reports/tasks.py#L26